CHANGELOG.md

```diff
@@ -24,7 +24,7 @@ Here is a template for new release sections
 ### Added
 -
 ### Changed
--
+- fix xlrd to xlrd==1.2.0 in requirements/default.txt (#716)
```

Comment:
maybe you can add a hint on the reason which lead to freezing the version number?

Reply:
I meant hint in the changelog entry :) so that someone reading this in 4 month and wondering why we frooze the version gets an answer immediately. Don't worry, I'll add it in my next PR if I don't forget :)

Reply:
ah ok, didn't gat that ;)

Reply:
This will be added in #722